clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -602,6 +602,13 @@ protected void reportPushStatus(
       int partition,
       ExecutionStatus status,
       Optional<String> incrementalPushVersion) {
+    if (hasCurrentVersionBootstrapping()) {
+      LOGGER.info(
+          "DaVinci is still bootstrapping, so skip the status report for store version: " + kafkaTopic + ", partition: "
+              + partition + ", status: " + status
+              + (incrementalPushVersion.isPresent() ? ", inc push version: " + incrementalPushVersion.get() : ""));
```

Comment:
Wait, sorry, I just saw this at the last moment after submitting my previous review... can we use string interpolation here like in the rest of the code, instead of concatenation?